### ISSUE TITLE
Please also consider accessibility for the visually impaired.

### ISSUE THREAD

**Issue by kbuwel** [Anjok07/ultimatevocalremovergui#862]
I am visually impaired. 
The contents of the screen are read using a screen reader. 
Ultimate Vocal Remover is a really good tool, but I'm having trouble downloading the model. 
It is difficult to understand the contents of the screen using a screen reader. 
When I try to download a model, the screen reader cannot read the download button and does not 
properly provide me with information about which one to select. 
Even if you download the model, it is difficult to remove music with Ultimate Vocal Remover because 
it does not give you correct information. 
I hope this is resolved in the next version.

**Comment by cn-peter** [comment]
视障人路过。
